types/stripe-js/checkout.d.ts

```diff
@@ -22,6 +22,24 @@ declare module '@stripe/stripe-js' {
      * An array of objects representing the items that your customer would like to purchase.
      * These items are shown as line items in the Checkout interface and make up the total amount to be collected by Checkout.
      */
+    lineItems?: Array<{
+      /**
+       * The ID of the price that the customer would like to purchase. SKU or plan IDs may also be used.
```

Comment:
That JSdoc comment seems wrong.

Reply:
How so? It's also what's documented in the js ref.

Reply:
Sorry, bad brain. It just looked weird next to `items`